edivorce/apps/core/static/js/functions.js

```diff
@@ -355,6 +355,30 @@ var validateEmail = function(el){
     }
 };
 
+var validateName = function(el){
+
+    console.log('validating name')
```

Comment:
Can you clean up test code?

Reply:
Whoops, missed that one, thanks. Updated in the latest commit.